src/plugins/workspace/public/components/delete_workspace_modal/show_delete_details_modal.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiSmallButton,
+  EuiModal,
+  EuiModalBody,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCommentList,
+  EuiCommentProps,
+  EuiBadge,
+  EuiModalFooter,
+} from '@elastic/eui';
+import { CoreStart, WorkspaceAttribute } from 'opensearch-dashboards/public';
+import { i18n } from '@osd/i18n';
+
+export interface ShowDeleteDetailsModalProps {
+  selectedWorkspaces: WorkspaceAttribute[];
+  failedWorksapces: WorkspaceAttribute[];
+  openModal: (
+    node: React.ReactNode,
+    options?: Parameters<CoreStart['overlays']['openModal']>['1']
+  ) => ReturnType<CoreStart['overlays']['openModal']>;
+}
+
+export const ShowDeleteDetailsModal = (
```

Comment:
I'm prefer to renaming to `showDeleteDetailsModal`. Start with upper case more like a class or a component.